### DISCUSSION TITLE
Why is the api page still showing v1 version after I install API-2.0

### DISCUSSION THREAD

**Discussion by ouyoung33** [jaredhendrickson13/pfsense-api#433] [Q&A]
![image](https://github.com/jaredhendrickson13/pfsense-api/assets/128574646/f96cde1f-ad25-4b75-a4bb-0e12e01f68b3)

**Comment by jaredhendrickson13** [comment]
Do you have both v1 and v2 installed? The settings and documentation pages for v2 are now within the System > REST API page. System > API accesses settings and documentation for v1.

**Reply by ouyoung33** [reply]
It has been solved, we need to change the link address in the document to: pkg -C /dev/null   https://github.com/jaredhendrickson13/pfsense-api/releases/download/v2.0.0-dev-cbc8ea0/pfSense-2.7.2-pkg-RESTAPI.pkg  .Thank you very much.
